ui/app/(ai)/analyst/(tools)/checks.ts

```diff
@@ -0,0 +1,17 @@
+import { tool } from "@langchain/core/tools";
+
+import { aiGetProviderChecks } from "@/lib/lighthouse/helperChecks";
+import { checkSchema } from "@/types/lighthouse";
+
+export const getProviderChecksTool = tool(
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!

Reply:
Moved all tools code to `/lib/lighthouse/tools/` directory